package com.itguigu.TestException_;

/**
 * @author z
 * @create 2020-07-10-21:29
 */
public class TestException_1 {
    public static void main(String[] args) {
        int arr[] = new int[5];
        arr[10]=7;
        System.out.println("end of main() method!!");
    }
}
